RacingCar/controller/Game.swift

```diff
@@ -11,8 +11,8 @@ class Game {
     private let cars: [Car]
     private let counts: Int
```

Comment:
```suggestion
    private let count: Int
```
counts라는 이름이 [Int]을 연상하게 하는데 count 는 어떨까요? 👀